ui/src/app/utils/runtime-utils.tsx

```diff
@@ -239,7 +239,7 @@ const useRuntime = (currentWorkspaceNamespace) => {
 
 export const maybeInitializeRuntime = async(workspaceNamespace: string, signal: AbortSignal): Promise<Runtime> => {
   if (workspaceNamespace in compoundRuntimeOpStore.get()) {
-    await new Promise((resolve, reject) => {
+    await new Promise<void>((resolve, reject) => {
```

Comment:
Resolved `error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?`